sdk/core/System.ClientModel/README.md

```diff
@@ -25,67 +24,144 @@ None needed for `System.ClientModel`.
 
 ### Authenticate the client
```

Comment:
Can we remove this section ("Authenticate the client"). I think it's intended for client libraries.